easybuild/easyconfigs/p/PsiCLASS/PsiCLASS-1.0.3-foss-2021b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = "MakeCp"
+
+name = "PsiCLASS"
+version = "1.0.3"
+
+homepage = "https://github.com/splicebox/PsiCLASS"
+description = """PsiCLASS is a reference-based transcriptome assembler for single or multiple RNA-seq samples."""
+
+toolchain = {"name": "foss", "version": "2021b"}
+
+source_urls = ["https://github.com/splicebox/PsiCLASS/archive/"]
+sources = ["v%(version)s.tar.gz"]
+patches = ["PsiCLASS-samtool_as_dep.patch"]
+
+dependencies = [
+    ("SAMtools", "0.1.19"),
+]
+
+files_to_copy = ["*"]
```

Comment:
this will copy the bundled `samtools` directory to the install dir as well, I think we would want to avoid that